ibmdbpy/frame.py

```diff
@@ -2316,3 +2323,106 @@ def check_numeric_columns(idaobject):
         check_numeric_columns(self)
         if isinstance(other, ibmdbpy.IdaSeries)|isinstance(other, IdaDataFrame):
             check_numeric_columns(other)
+
+###############################################################################
+### New functionalities for IdaDataFrames -- Experimental
+###############################################################################
+    
+    def delete_na(self, columns, logic = "any", inplace = False):
+        """
+        
+        Parameters
+        -------
+        columns: list of eligible column names
```

Comment:
Please use an indentation style that's more like other functions in this file (for example https://github.com/ibmdbanalytics/ibmdbpy/blob/c5750cf8961fc0555af8bfb62c65116f1c5c1c0b/ibmdbpy/frame.py#L1261-L1274):

```suggestion
        columns: list of str
            list of eligible column names
```

This separates the separate parameters visually.
```

Reply:
ok done